src/App.jsx

```diff
@@ -1,14 +1,38 @@
+/* eslint-disable no-lone-blocks */
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* eslint-disable prettier/prettier */
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+import { PostList } from './components/PostList';
```

Comment:
The `PostList` component is used in the JSX, so this import is necessary. Ensure that the component is correctly implemented in the `./components/PostList` file.